lib/test_track_rails_client/engine.rb

```diff
@@ -1,19 +1,5 @@
-require 'delayed_job'
-
-begin
-  require 'airbrake'
-rescue LoadError
-  # no-op
-end
-
-unless defined?(Delayed::Plugins::Airbrake) && Delayed::Worker.plugins.include?(Delayed::Plugins::Airbrake)
-  begin
-    require 'delayed-plugins-airbrake'
-  rescue LoadError
-    # no-op
-  end
-end
```

Comment:
i'm totally fine with removing this. we bundled it by default for a bit then when we stopped using airbrake internally we needed to make it optional and we ended up here.